src/web/containers.cc

```diff
@@ -86,7 +86,7 @@ void Web::Containers::process()
 #ifdef HAVE_INOTIFY
             if (config->getBoolOption(CFG_IMPORT_AUTOSCAN_USE_INOTIFY)) {
                 auto adir = database->getAutoscanDirectory(cont->getID());
-                if (adir && (adir->getScanMode() == ScanMode::INotify))
+                if (adir && (adir->getScanMode() == AutoscanDirectory::ScanMode::INotify))
                     autoscanMode = "inotify";
```

Comment:
Shouldn't we call `AutoscanDirectory::mapScanmode` here as well

Reply:
Yes, good idea.

Reply:
Actually, no this is mapping the CDSObjects autoscan field, not an autoscan itself, its still just a bunch of #defines

I am hoping to clean this up when collapsing the multiple "AutoscanList" into one